lib/manticore/patches/client.rb

```diff
@@ -0,0 +1,21 @@
+module Manticore
+  class Client
```

Comment:
If possible, I would prefer we create a module to contain our patch, and to mix that module into `Manticore::Client` (instead of monkey-patching `Manticore::Client` directly). I would also prefer for that module to be scoped within the plugin, so if we have to chase it down from some other plugin's use of manticore, we can see the patch on the client's ancestor chain.